code/modules/clothing/clothing.dm

```diff
@@ -53,8 +54,9 @@
 
 	if(!M.incapacitated() && loc == M && istype(over_object, /obj/screen/inventory/hand))
 		var/obj/screen/inventory/hand/H = over_object
-		if(M.putItemFromInventoryInHandIfPossible(src, H.held_index))
-			add_fingerprint(usr)
+		if(mouse_unequippable)
+			if(M.putItemFromInventoryInHandIfPossible(src, H.held_index))
+				add_fingerprint(usr)

```

Comment:
condense the ifs

Reply:
wouldn't that cause `M.putItemFromInventoryInHandIfPossible` to be called in the if statement? that would break it.

Reply:
nvm TIL short-circuiting